WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -812,24 +812,26 @@ private extension ProductFormViewController {
 //
 private extension ProductFormViewController {
     func editProductType() {
-        //TODO-2198: present the bottom action sheet
-    }
-
-    func onEditProdyctTypeCompletion(productType: ProductType) {
-        defer {
-            navigationController?.popViewController(animated: true)
-        }
-        let hasChangedData: Bool = {
-            productType != product.productType
-        }()
+        let title = NSLocalizedString("Select a product type",
+                                      comment: "Message title for select a product type action bottom sheet")
```

Comment:
super nit: maybe "Message title of bottom sheet for selecting a product type"?

Reply:
Done 👍 94bcdd0a4a08b5fad05ef9dc7a36eacea4254ab6